scripts/e2e-test.sh

```diff
@@ -78,3 +78,29 @@ helm template deploy/helm/eunomia-operator/ \
   --set eunomia.operator.image.tag=dev \
   --set eunomia.operator.namespace=$TEST_NAMESPACE | kubectl delete -f -
 
+# Below block is to ensure hello-world-yaml example is working
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never \
+  --set eunomia.openshift.route.enabled=true | kubectl apply -f -
+
+kubectl create namespace eunomia-hello-world-yaml-demo
+
+kubectl apply -f examples/hello-world-yaml/eunomia-runner-sa.yaml -n eunomia-hello-world-yaml-demo
+
+kubectl apply -f examples/hello-world-yaml/cr/hello-world-cr1.yaml -n eunomia-hello-world-yaml-demo
+
+sleep 30s
+
+RESULT=`kubectl get po -n eunomia-hello-world-yaml-demo -l name=hello-world -o=jsonpath="{range .items[*]}{.status.phase}{'\n'}{end}"`
+
+if [[ $RESULT == "Running" ]]
+then
+  echo "Example Passed"
```

Comment:
in the hello-world-yaml example, there is also ...cr2.yaml and ...cr3.yaml, why are you not checking those, but you say "Example Passed"?

Reply:
Ok, i will add steps validate cr2 and cr3 as well 👍